ee/clickhouse/queries/experiments/secondary_experiment_result.py

```diff
@@ -29,16 +30,27 @@ def __init__(
     ):
 
         breakdown_key = f"$feature/{feature_flag.key}"
-        variants = [variant["key"] for variant in feature_flag.variants]
+        self.variants = [variant["key"] for variant in feature_flag.variants]
+
+        # our filters assume that the given time ranges are in the project timezone.
+        # while start and end date are in UTC.
+        # so we need to convert them to the project timezone
+        if team.timezone:
+            start_date_in_project_timezone = experiment_start_date.astimezone(pytz.timezone(team.timezone))
+            end_date_in_project_timezone = (
+                experiment_end_date.astimezone(pytz.timezone(team.timezone)) if experiment_end_date else None
+            )
 
         query_filter = filter.with_data(
             {
-                "date_from": experiment_start_date,
-                "date_to": experiment_end_date,
+                "date_from": start_date_in_project_timezone,
+                "date_to": end_date_in_project_timezone,
+                "explicit_date": True,
                 "breakdown": breakdown_key,
                 "breakdown_type": "event",
-                "properties": [{"key": breakdown_key, "value": variants, "operator": "exact", "type": "event"}],
-                # :TRICKY: We don't use properties set on filters, instead using experiment variant options
+                "properties": [],
+                # :TRICKY: We don't use properties set on filters, as these
```

Comment:
Didn't fully understand this

Reply:
A bit annoying, during creation, the properties are what's supposed to be the flag filters, not the metric filter.

We haven't gotten around to changing the creation flow, so this still exists.

Reply:
(same comment as in https://github.com/PostHog/posthog/blob/master/ee/clickhouse/queries/experiments/funnel_experiment_result.py#L80 )